packages/graph/hash_graph/lib/graph/src/store/mod.rs

```diff
@@ -30,4 +33,19 @@ pub use self::{
 ///
 /// In addition to the errors described in the methods of this trait, further errors might also be
 /// raised depending on the implementation, e.g. connection issues.
-pub trait Store = AccountStore + DataTypeStore + PropertyTypeStore + EntityTypeStore + EntityStore;
+#[async_trait]
+pub trait Store:
+    AccountStore + DataTypeStore + PropertyTypeStore + EntityTypeStore + EntityStore
+{
+    type Transaction<'t>: Transaction + Send
```

Comment:
`Send` is required for utoipa and we probably don't want to add `Send` at every location

Reply:
Can you add a comment explaining this

Reply:
Cannot say why it failed before but it's not needed 🤷🏻‍♂️ f058e1b938ed2e0d41709baef0c9e7dfc5c06af3